### PR TITLE
docs: add research docs

### PR DESCRIPTION
**Pull request by gpsanant**

### CODE REVIEW

File: docs/releases/operator-set-release/avs-considerations.md [hunk 1]
```diff
@@ -0,0 +1,84 @@
+# AVS Considerations
+
+The operatorSet release solves 3 painpoints for AVSs:
+
+1. Quorums cannot be displayed on the UI with their proper TVL and strategies
+2. MetaAVSs are not supported
+3. Quorums cannot properly be rewarded for the in the current rewards released
+4. Configurable operator commissions per AVS/operatorSet
```
Comment (samlaf) [inline docs/releases/operator-set-release/avs-considerations.md:8]: says 3 painpoints but there's 4 bulletpoints.
Also (2) is a bit confusing. Is this saying that metaavss not being supported is a painpoint that is being solved, or is it saying that we are solving this previous painpoint. I would rephrase.

**Comment by ypatil12** [comment]
Stale
